### ISSUE TITLE
formatter: colorsEnabled: true not honoured

### ISSUE THREAD

**Issue by davidjgoss** [cucumber/cucumber-js#2011]
### 👓 What did you see?

When outputting to a non-TTY environment, using `colorsEnabled: true` in the format options doesn't result in colours being output.

### ✅ What did you expect to see?

Colours to be output, because of this logic:

```ts
if (doesNotHaveValue(configuration.options.colorsEnabled)) {
      typeOptions.parsedArgvOptions.colorsEnabled = (
        stream as TtyWriteStream
      ).isTTY
    }
```

In other words, if we specify the option, it should output colours without checking if the stream is TTY.

### 📦 Which tool/library version are you using?

cucumber-js 8.1.2

### 🔬 How could we reproduce it?

TBA

**Comment by davidjgoss** [comment]
@binarymist I've raised this as the place to discuss your issue from https://github.com/cucumber/cucumber-js/issues/1711#issuecomment-1110254136.

Is it right that you're running cucumber-js on a server environment and then presenting the output to users elsewhere? So you add `colorsEnabled: true` to explicitly enable coloured output.

For 8.0.0 we switched libraries from `colors` to `chalk` for well-known reasons, but I notice that chalk does auto detection of colours support and noops if it doesn't find it https://github.com/chalk/chalk#supportscolor so we may need to make some changes in our integration there to force it to honour our option. In the meantime if it's possible, could you try setting `FORCE_COLOR=1` as an env variable where you run cucumber-js?

cc @aurelien-reeves

**Comment by binarymist** [comment]
Hi @davidjgoss and thanks!

> Am I right that you're running cucumber-js on a server environment and then presenting the output to users elsewhere? So you add colorsEnabled: true to explicitly enable coloured output.

Yes. the CLI text output with colour codes is sent to Redis which lands in the _orchestrator_ service. The _orchestrator_ then sends the text from Redis to the _CLI_ which is on the end users machine.

Also note we're still on "@cucumber/cucumber": "^8.0.0-rc.3"

Will try `FORCE_COLOR=1` when we get a chance.

**Comment by binarymist** [comment]
`FORCE_COLOR=1` does enable colours. What's your strategy from here @davidjgoss ?

**Comment by aurelien-reeves** [comment]
You can look at https://github.com/cucumber/cucumber-js/pull/2026 to see what is going on

**Comment by davidjgoss** [comment]
This is released in 8.2.0, see the updated doc: https://github.com/cucumber/cucumber-js/blob/main/docs/formatters.md#colored-output
